docs_src/design/adr/013-Device-Service-Events-Message-Bus.md

```diff
@@ -0,0 +1,259 @@
+
+Receive only Events from the `Random-Integer-Device` and `Random-Boolean-Device` profiles
+
+```toml
+[Binding]
+Type="messagebus"
+SubscribeTopics="edgex/events/Random-Integer-Device, edgex/events/Random-Boolean-Device"
+```
+Receive only Events from the  `Random-Integer-Device1` from the `Random-Integer-Device` profile
+
+```toml
+[Binding]
+Type="messagebus"
+SubscribeTopics="edgex/events/Random-Integer-Device/Random-Integer-Device1"
+```
+
+or receives all Events:
+
+```toml
+[Binding]
+Type="messagebus"
+SubscribeTopics="edgex/events/#"
+```
+
+### Secure Connections
+
+As stated earlier,  this ADR is dependent on the  [**Secret Provider for All**](TBD) ADR to provide a common Secret Provider for all Edgex Services to access their secrets. Once this is available, the Message Bus connection can be secured via the following configurable authentications modes which follows the implementation for secure MQTT Export and secure MQTT Trigger used in Application Services.
+
+- **none** - No authentication used
+- **usernamepassword** - Username & password authentication. CA Cert used in conjunction if it exists in the Secret Provider. 
+- **clientcert** - Client certificate and key for authentication. CA Cert used in conjunction if it exists in the Secret Provider. 
+- **cacert** - CA Certificate only, i.e. just a TLS enabled connection
+
+The secrets specified for the above options are pulled from the `Secret Provider` using the configured `SecretPath`. If the `cacert` secret exists along with the secrets for one of the other modes, it will be used in conjunction to enable a TLS connection in addition to the other mode.
+
+How the secrets are injected into the `Secret Provider` is out of scope for this ADR and covered in the [**Secret Provider for All**](TBD) ADR. 
+
+> *Note: It has been decide the TLS connections are not required for on box service to service communications. This includes Device Service to Broker to App Service. The configuration will still support certificates for those deployments that feel certificates are needed, but will not be used by default EdgeX deployment.*
+
+## Consequences
+
+- If C SDK doesn't support `ZMQ` or `Redis Streams` then there must be a MQTT Broker running when a C Device service is in use and configured to publish to Message Bus.
+- If we adopt proposed publish topic scheme with `DeviceProfileName` and `DeviceName` the V2 API must restrict the characters used in device names to those allowed in a topic.  An [issue](https://github.com/edgexfoundry/go-mod-core-contracts/issues/343) for V2 API already exists for restricting the allowable characters to [RFC 3986](https://tools.ietf.org/html/rfc3986) , which will suffice.
+- Newer ZMQ may allow for multiple publishers. Requires investigation and very likely rework of the ZMQ implementation in go-mod-messaging.
```

Comment:
I've done a bit of investigation and haven't found anything concrete yet. From what I've read the only options are:

 * each publisher gets its own port
 * you implement a ZMQ broker